shoreline.go

```diff
@@ -39,6 +43,30 @@ type (
 	}
 )
 
+func kafkaSender() *kafka_sarama.Sender {
```

Comment:
This function combines two distinct actions.  The first action is to get the configuration of the sender from the environment.  The second action is to create a sender from a configuration.

I recommend splitting these two actions into two separate functions.  This will make it more easily testable.

Reply:
done